src/components/Cardlist/index.tsx

```diff
@@ -0,0 +1,27 @@
+import React, { FC } from "react";
+import Card from "../Card";
+import styles from "./index.module.css";
+
+interface IProps {
+  data: {
+    body_text: string;
+    header_text: string;
+    id: number;
+  }[];
+}
+
+const CardList: FC<IProps> = ({ data }) => {
+  return (
+    <div className={styles.card_list}>
```

Comment:
give better naming to things, what is data? what is e? you can write Cards, and eachCard respectively